features/built_in_matchers/include.feature

```diff
@@ -1,7 +1,8 @@
 Feature: `include` matcher
 
-  Use the `include` matcher to specify that a collection includes one or more expected objects.
-  This works on any object that responds to `#include?` (such as a string or array):
+  Use the `include` matcher to specify that a collection includes one or more
+  expected objects. You can also think of it as an `any`-matcher. This works on
+  any object that responds to `#include?` (such as a string or array):
```

Comment:
Same here -- please don't wrap this.  Make it one long line so it renders better on relish.